livekit-agents/livekit/agents/voice_assistant/assistant.py

```diff
@@ -685,6 +708,12 @@ async def _synthesize_task(
             _first_frame = True
             async for audio in self._tts.synthesize(data.source):
                 if _first_frame:
+                    if self._agent_transcription_manager:
```

Comment:
This isn't the right place to do it, this should maybe be inside the maybe_play_speech and only triggered if validated channel is closed